### PR TITLE
MAGECLOUD-3665: Unable to install 2.2.6 with ece-tools develop after shell refactoring

### PR DESCRIPTION
**Pull request by oshmyheliuk**
### Description

Fixed bug after shell refactoring.
There are different logic in symfony/process packages between 2.x and 4.x. Version 4.x can accept `commandLine` as an array but in 2.x it has to be a string.

### Fixed Issues (if relevant)

https://magento2.atlassian.net/browse/MAGECLOUD-3665

### Manual testing scenarios

1. Install magento 2.2.6 on cloud with configure post-deploy hook
2. Check that there are no fatal errors during post-deploy phase

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Review by bbatsche — DISMISSED** [review]

**Review by shiftedreality — DISMISSED** [review]

**Comment by andriyShevtsov** [comment]
QA approved

File: src/Shell/Shell.php [hunk 1]
```diff
@@ -63,12 +63,16 @@ public function __construct(
      * ```php
      * $this->shell->execute('/bin/bash -c "set -o pipefail; firstCommand | secondCommand"');
      * ```
+     *
+     *  `commandline` should be always a string as symfony/process package v2.x doesn't support array-type `commandLine`
      */
     public function execute(string $command, array $args = []): ProcessInterface
     {
         try {
+            $args = array_map('escapeshellarg', array_filter($args));
```
Comment (NadiyaS) [inline src/Shell/Shell.php]: why _array_filter()_ is used here?

Comment (oshmyheliuk) [reply src/Shell/Shell.php]: I've removed `array_filter()` as argument can be `false` or `0`

**Review by NadiyaS — APPROVED** [review]

**Review by shiftedreality — APPROVED** [review]

**Comment by andriyShevtsov** [comment]
QA approved

**Pull request merged**